components/micro-gateway-cli/src/main/resources/templates/policy.mustache

```diff
@@ -1,54 +1,19 @@
 import wso2/gateway;
 
-stream<gateway:IntermediateStream> s{{name}}intermediateStream = new;
-stream<gateway:GlobalThrottleStreamDTO> s{{name}}resultStream = new;
-stream<gateway:EligibilityStreamDTO> s{{name}}eligibilityStream = new;
-stream<gateway:RequestStreamDTO> s{{name}}reqCopy= gateway:getRequestStream();
-stream<gateway:GlobalThrottleStreamDTO> s{{name}}globalThrotCopy = gateway:getGlobalThrottleStream();
+stream<gateway:InputRequest> s{{name}}inputStream =gateway:getinputStream();
```

Comment:
Can't we use the request stream and gateway:RequestStreamDTO instead of defining new streams and dtos